### PR TITLE
SUBS-519 Thanks Page Design Changes

### PR DESCRIPTION
**Pull request by eddieferrer**
* Design and layout changes to thanks page.
* These design changes are in preparation for the CTA in SUBS-519 but were decided to be made permanent and not part of the proposed experiment.

### CODE REVIEW

**Comment by eddieferrer** [comment]
screencapture-dev-vm-01-kiva-org-checkout-thanks-2020-10-14-08_05_30

File: src/components/Checkout/SocialShare.vue [hunk 1]
```diff
@@ -174,7 +182,7 @@ export default {
 		},
 		suggestedMessage() {
 			if (this.selectedLoan.name) {
-				const location = _get(this, 'selectedLoan.geocode.city') || _get(this, 'selectedLoan.geocode.country.name'); // eslint-disable-line max-len
+				const location = this.selectedLoan?.geocode?.city || this.selectedLoan?.geocide?.country?.name;
```
Comment (mcstover) [inline src/components/Checkout/SocialShare.vue:185]: Spelling issue: `geocide`

File: src/pages/Thanks/ThanksPage.vue [hunk 2]
```diff
@@ -116,13 +150,13 @@ export default {
 			// The default empty object and the v-if will prevent the
 			// receipt from rendering in the rare cases this query fails.
 			// But it will not throw a server error.
-			this.receipt = _get(data, 'shop.receipt');
-			const loansResponse = _get(this.receipt, 'items.values', []);
+			this.receipt = data?.shop?.receipt;
+			const loansResponse = this.receipt?.items?.values || [];
```
Comment (mcstover) [inline src/pages/Thanks/ThanksPage.vue:154]: I've been using the "nullish" `??` in these cases.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_Coalescing_Operator

File: src/components/Checkout/SocialShare.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/Checkout/SocialShare.vue:185]: Yikes! Good catch

**Review by mcstover — APPROVED** [review]
This is a nice layout update!

File: src/pages/Thanks/ThanksPage.vue (hunk 2 above)
Comment (mcstover) [reply src/pages/Thanks/ThanksPage.vue]: Def would like to hear your thoughts on this operator there's likely other locations we'll want to update.

**Review by BoulderBrains — COMMENTED** [review]
👀

**Pull request merged**